services/migrations/github.go

```diff
@@ -872,3 +872,15 @@ func (g *GithubDownloaderV3) GetReviews(ctx context.Context, reviewable base.Rev
 	}
 	return allReviews, nil
 }
+
+// FormatCloneURL add authentication into remote URLs
+func (g *GithubDownloaderV3) FormatCloneURL(opts MigrateOptions, remoteAddr string) (string, error) {
+	u, err := url.Parse(remoteAddr)
+	if err != nil {
+		return "", err
+	}
+	if len(opts.AuthToken) > 0 {
+		u.User = url.UserPassword("oauth2", strings.Split(opts.AuthToken, ",")[0])
```

Comment:
What about this comment?

```suggestion
		// "multiple tokens" are used to benefit more "API rate limit quota"
		// git clone doesn't count for rate limits, so only use the first token.
		u.User = url.UserPassword("oauth2", strings.Split(opts.AuthToken, ",")[0])
```